### ISSUE TITLE
Link accessibility review

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1394]
Per "relying solely on color for links," we should review link styles, particularly the general ones in body text.

Reviewing link accessibility and found this from 18F:

>Links that only rely on color also fail this requirement. Links must be distinguished by more than just color (hue). Links can be distinguished by underlines, symbols, context (in a menu for example), or lightness. If the contrast between the surrounding text and a link is greater than 3:1 it is acceptable. Also note, the link text itself still needs to have a contrast ratio of 4.5:1 or greater with the background. More information about link color dependence.

And W3C: https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F73
